marketplace/src/styles.scss

```diff
@@ -177,8 +176,8 @@ app-api-documentation .mat-expansion-panel-body {
 .swagger-ui .model-toggle {
   top: 0px;
   vertical-align: middle;
-  filter: invert(99%) sepia(0%) saturate(416%) hue-rotate(314deg)
-    brightness(116%) contrast(100%);
+  //filter: invert(99%) sepia(0%) saturate(416%) hue-rotate(314deg)
```

Comment:
Hier ist nochmal auskommentierter Code. Außerdem kann glaube ich noch der Code aus den Zeilen 63ff. 128ff. und 208ff. weg

Reply:
Das habe ich noch dagelassen für Jonas wenn er an die Swagger Styles geht, damit er noch Anhaltspunkte hat

Reply:
Teile davon sind noch von der Seitennavigation. Bitte in dem Branch wo du die Tests nachreichst korrigieren